dassh/correlations/properties_NaK.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+class mat_from_corr(object):
+    """
+    Correlation object for sodium properties
+    
+    Parameters
+    ----------
+    prop : str
+        Property name to be calculated
+        
+    Returns
+    ----------
+    float
+        Property value
+        
+    Notes
+    ----------
+    Reference correlations can be found in:
+    [1] O. J. Foust, Ed., Sodium-NaK Engineering Handbook 
+    Volume I - Sodium Chemistry and Physical Properties, 
+    Gordon and Breach, Science Publishers, Inc., 1972.
+    """
+    def __init__(self, prop):
+        self.prop = prop
+    
+    def __call__(self, temperature):
+        return getattr(self, self.prop)(temperature)
+    
+    def density(self, T):
```

Comment:
Please, add the docstring at least for each public method, providing the identifier for the corresponding correlation in the referenced work [1]